content/shmem_request_merge.tex

```diff
@@ -0,0 +1,29 @@
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void shmem_request_merge(shmem_request_t *req, int count, shmem_request_t[] *requests);
+\end{Csynopsis}
```

Comment:
I think we should not propose the function interface, yet. I don’t see a clear motivation. 
It seems more for completion and usability. What do you think ?